setup.py

```diff
@@ -2,12 +2,15 @@
 
 try:
     from setuptools import setup, Extension
+    from setuptolls.command.build_ext import build_ext as base_build_ext
```

Comment:
This should be `setuptools`, no? (Not `tolls`...)

Reply:
yikes, yes, im going to put in a warning with the exception when it fails